### PR TITLE
Make go get compatible

### PR DESCRIPTION
**Pull request by stevenh**
Added +build windows to all existing source files.

Added go-get.go which compiles for !windows so there is at least one file to compile.

The one line other change was due to go fmt hence its a no-op.

### CODE REVIEW

File: wmi.go [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-		q := wmi.CreateQuery(&dst, "")
-		err := wmi.Query(q, &dst)
-		if err != nil {
-			log.Fatal(err)
-		}
-		for i, v := range dst {
-			println(i, v.Name)
-		}
-	}
+// Package wmi provides a WQL interface for WMI on Windows.
```
Comment (mjibson) [inline wmi.go]: Why the change in comment style?

File: go-get.go [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// This file is here so go get succeeds, as without it errors with:
+// no buildable Go source files in ...
+//
+// +build !windows
+
+package wmi
```
Comment (mjibson) [inline go-get.go]: I'm not sure why this is necessary. I'm ok with those messages, since they aren't errors, just warnings, and go get continues.

File: wmi.go (hunk 1 above)
Comment (stevenh) [inline wmi.go]: Without the comment change the build directive isn't processed
unfortunately.


On 27 Jan 2015, at 08:18, Matt Jibson <notifications@github.com> wrote:

In wmi.go <https://github.com/StackExchange/wmi/pull/7#discussion_r23592541>
:

> -	type Win32_Process struct {
> -		Name string
> -	}
> -
> -	func main() {
> -		var dst []Win32_Process
> -		q := wmi.CreateQuery(&dst, "")
> -		err := wmi.Query(q, &dst)
> -		if err != nil {
> -			log.Fatal(err)
> -		}
> -		for i, v := range dst {
> -			println(i, v.Name)
> -		}
> -	}
> +// Package wmi provides a WQL interface for WMI on Windows.

Why the change in comment style?

—
Reply to this email directly or view it on GitHub
<https://github.com/StackExchange/wmi/pull/7/files#r23592541>.

File: go-get.go (hunk 2 above)
Comment (stevenh) [inline go-get.go]: When I tested that was not the case the go get aborted at this error.


On 27 Jan 2015, at 08:19, Matt Jibson <notifications@github.com> wrote:

In go-get.go
<https://github.com/StackExchange/wmi/pull/7#discussion_r23592564>:

> @@ -0,0 +1,6 @@
> +// This file is here so go get succeeds, as without it errors with:
> +// no buildable Go source files in ...
> +//
> +// +build !windows
> +
> +package wmi

I'm not sure why this is necessary. I'm ok with those messages, since they
aren't errors, just warnings, and go get continues.

—
Reply to this email directly or view it on GitHub
<https://github.com/StackExchange/wmi/pull/7/files#r23592564>.

Comment (mjibson) [inline go-get.go]: Well, it won't `go get` any dependencies of *this* project (obviously, since there was nothing to compile), but it will continue with other dependencies of other packages.

File: wmi.go (hunk 1 above)
Comment (mjibson) [inline wmi.go]: Then the package doc needs to be moved so the build tag can be separate.
